src/TelluricFitter.py

```diff
@@ -626,7 +627,8 @@ def GenerateModel(self, pars, nofit=False, separate_source=False, return_resolut
                         Ignored if nofit=False
         :param model: A DataStructures.xypoint instance containing an un-broadened telluric model.
                       If given, it uses this instead of making one.
-
+        :param air_wave: Set True of False to overwrite the air_wave saved under self. Default is None.
```

Comment:
Add to the docstring: "Default is None, which will use the `self.air_wave` attribute"